backend/src/main/java/com/bakdata/conquery/models/events/CBlock.java

```diff
@@ -43,8 +43,7 @@
 	 * Statistic for fast lookup if entity is of interest.
 	 * Int array for memory performance.
 	 */
-	private int[] minDate;
-	private int[] maxDate;
+	private int[] minDate, maxDate;
```

Comment:
In C ist das nicht gerne gesehen, bei Java kann ich mir sowas auch vorstellen, weil man sowas produzieren kann:
`private int minDate, maxDate [];` oder `private int [] minDate, maxDate;`

Dann ist auf den ersten Blick nicht ganz klar was gemeint ist, wenn nicht eine Zeile pro Deklaration verwendet wird.